README.org

```diff
@@ -68,12 +72,18 @@ export DB_PASSWORD=mypass
 export DB_HOST=127.0.0.1
 export DB_PORT=5432
 python3 ./grinexplorer/manage.py migrate
-python3 ./grinexplorer/manage.py runserver
+python3 ./grinexplorer/manage.py runserver <IP address:port>
 #+end_src
 
-*** Import blockchain
+*** You would have issue TemplateDoesNotExist at / base.html 
+Try to copy  ~/grin-explorer/grinexplorer/templates/base.html to  ~/grin-explorer/templates/ and other required directories.
```

Comment:
Thank you for the improvements, but this shouldn't be necessary. The template directories are properly setup in the config file.